tests/README.md

```diff
@@ -1 +1,21 @@
-Until we have actual tests, this is a placeholder so Git keeps the directory.
+# Phyloreferencing Test Suite
+
+## Usage
+> py.test tests/
+
+## Description
+This test suite is designed to be used to validate the outputs of the `phylo2owl.py` script. It does this in two ways:
+
+ 1. By comparing an output file with its expected output, and reporting an error if they differ.
+ 2. By validating an output OWL file against a set of SHACL shapes.
+
+You can examine [previous runs of this test suite](https://travis-ci.org/phyloref/phylo2owl) on Travis-CI.
+
+## Test scripts
+
+ - `conftest.py`: A py.test configuration file that sets up the fixtures.
```

Comment:
If it's setting up the fixtures, shouldn't it be named `fixtures.py`, or at least `conf_test.py`? At first glance it reads as a test of configuration. Renaming to `fixtures.py` or some such would most easily prevent that mental mistake.